pyuvdata/tests/test_mwa_corr_fits.py

```diff
@@ -294,3 +294,64 @@ def test_diff_obs():
         mwa_uv.read([bad_obs, filelist[0], filelist[1]])
     assert str(cm.value).startswith('files from different observations')
     del(mwa_uv)
+
+
+@pytest.mark.filterwarnings("ignore:telescope_location is not set. ")
+@pytest.mark.filterwarnings("ignore:coarse channels are not contiguous for this observation")
+@pytest.mark.filterwarnings("ignore:some coarse channel files were not submitted")
+def test_flag_init():
+    """
+    Test that routine MWA flagging works as intended.
+    """
+    spoof_file1 = os.path.join(DATA_PATH, 'test/spoof_01_00.fits')
+    spoof_file6 = os.path.join(DATA_PATH, 'test/spoof_06_00.fits')
+    # spoof box files of the appropriate size
+    with fits.open(filelist[1]) as mini1:
+        mini1[1].data = np.repeat(mini1[1].data, 8, axis=0)
+        extra_dat = np.copy(mini1[1].data)
+        for app_ind in range(2):
+            mini1.append(fits.ImageHDU(extra_dat))
+        mini1[2].header['MILLITIM'] = 500
+        mini1[2].header['TIME'] = mini1[1].header['TIME'] + 0.5
+        mini1[3].header['MILLITIM'] = 1000
+        mini1[3].header['TIME'] = mini1[1].header['TIME'] + 1
```

Comment:
header['TIME'] should be an integer. header['MILLITIM'] should be 0 or 500. Removing lines 315-316 (and 327-328) will give the desired result.

Reply:
I think when I excluded 315-316 and 327-328 there was a problem on read when line 492 of mwa_corr_fits.py is executed. It seems like that line expects every hdu to have a TIME line in the header.

Reply:
Oh, I see. The lines should be `mini[2].header['TIME'] = mini[1].header['TIME']` and `mini[3].header['MILLITIM'] = 0` respectively. TIME advances by whole seconds every other hdu, and nonzero MILLITIM accounts for the half seconds in between.

Reply:
Is this robust against the case where we average to 2s inside the correlator? We do that sometimes, I think 2014 data might be taken that way.

Reply:
I removed ` + 0.5` on lines 315, 327 and replaced 1000 with 0 on lines 316, 328